Fatras/include/ActsFatras/Kernel/Simulation.hpp

```diff
@@ -86,8 +93,7 @@ struct SingleParticleSimulation {
     using PropagatorOptions = Acts::PropagatorOptions<Actions, Abort>;
 
     // Construct per-call options.
-    PropagatorOptions options(geoCtx, magCtx,
-                              Acts::LoggerWrapper{*localLogger});
+    PropagatorOptions options(geoCtx, magCtx, loggerWrapper);
```

Comment:
I would prefer constructing `LoggerWrapper{*localLogger}` on the fly here.

Reply:
loggerWrapper is passed so that it is consistent when you construct simulation with a localLogger or an external loggerWrapper